src/api.ts

```diff
@@ -1,15 +1,15 @@
 import express from 'express';
 import { capture } from '@snapshot-labs/snapshot-sentry';
-import { parseQuery, resize, setHeader, getCacheKey } from './utils';
+import { parseQuery, resize, setHeader, getCacheKey, ResolverType } from './utils';
 import { set, get, streamToBuffer, clear } from './aws';
 import resolvers from './resolvers';
 import constants from './constants.json';
 import { rpcError, rpcSuccess } from './helpers/utils';
-import { lookupAddresses, resolveNames } from './addressResolvers';
+import { lookupAddresses, resolveNames, clearCache } from './addressResolvers';
 import lookupDomains from './lookupDomains';
 
 const router = express.Router();
-const TYPE_CONSTRAINTS = Object.keys(constants.resolvers).join('|');
+const TYPE_CONSTRAINTS = [...Object.keys(constants.resolvers), 'address', 'name'].join('|');
```

Comment:
I don't remember where we use `name` resolver 🤔

Reply:
It's for resolveName, when passing a domain name (like chaitu.eth), and return an address.

It's there, but not also sure where we use it